Autobahn.swift

```diff
@@ -1,36 +1,40 @@
 import Foundation
 import AutobahnDescription
 
-// let autobahn = Autobahn()
-// autoban.defaultTask = "release"
+enum Highway: String, AutobahnDescription.Highway {
+	case build, test, deploy, release
+}
 
+Autobahn(Highway.self)
```

Comment:
So when actually trying to run this `Autobahn.swift` file we get compiler errors because the compiler considers this Swift file a module named `Autobahn` and thinks this a method on the actual module :(  Here is the error message:
```
$ autobahn drive build
Autobahn v0.1.0
Autobahn.swift:4:43: error: no type named 'Highway' in module 'AutobahnDescription'
enum Highway: String, AutobahnDescription.Highway {
                                          ^
Autobahn.swift:8:9: error: cannot call value of non-function type 'module<Autobahn>'
Autobahn(Highway.self)
~~~~~~~~^
```

Reply:
Error on my part, this working great 👍

Reply:
I thought about naming this class AutobahnDescription like the module, but this is following the same naming convention like PackageDescription and Package. Still a bit confusing since there are two Autobahn classes now 😆